apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/FuzzySearch.tsx

```diff
@@ -56,31 +62,32 @@ class FuzzySearch extends PureComponent<FuzzySearchProps, FuzzySearchState> {
         pendingRequest: undefined,
     };
 
-    doSearch = (value: string) => {
-        if (!value) return;
-        const emoji = value.slice(0, 2);
-        const ident = value.slice(3).split(':');
+    doSearch = (option: SearchOption) => {
+        const result = option.result;
+        if (!result) return;
```

Comment:
nit: brace your conditions

```suggestion
        if (!result) {
        	return;
        }
```